styles/toolbox.css

```diff
@@ -0,0 +1,71 @@
+body {
+    background-color: #CDC0B0;
+    margin: 0 0 0 2px;
+}
+
+.flexbox {
+    display: flex;
+}
+
+/* default buttons have small text = 83% */
+.selector_button {
+    margin-top: 1px;
+    font-size: 16px;
```

Comment:
We should define font sizes in ems. I think it would be better if these were also bigger.